get-flavor.sh

```diff
@@ -2,7 +2,7 @@
 
 ID=$(lsb_release -i -s)
 if [ $? != 0 ] ; then echo "$0: lsb_release failed..."; exit 1; fi
-VERSION=$(lsb_release -r -s)
+VERSION=$(lsb_release -r -s | cut -d '.' -f 1)
```

Comment:
I'd rather leave VERSION be VERSION (and not MAJOR_VERSION) but use patternmatching for the case statement; that was actually on purpose at the time, because it was'nt clear if we'd care about differences for 6.7 or 7.1, but IMO it's clearer to use "7*" below if we don't